ZelBack/src/routes.js

```diff
@@ -337,7 +337,13 @@ module.exports = (app) => {
   app.get('/apps/globalappsspecifications/:hash?/:owner?/:appname?', cache('30 seconds'), (req, res) => {
     appsService.getGlobalAppsSpecifications(req, res);
   });
-  app.get('/apps/appspecifications/:appname?/:update?/:decrypt?', cache('30 seconds'), (req, res) => {
+  app.get('/apps/latestspecificationversion', cache('5 minutes'), (req, res) => {
+    appsService.getlatestApplicationSpecificationAPI(req, res);
+  });
+  app.get('/apps/appspecifications/:appname/update', cache('30 seconds'), (req, res) => {
+    appsService.updateApplicationSpecificationAPI(req, res);
+  });
+  app.get('/apps/appspecifications/:appname/:decrypt?', cache('30 seconds'), (req, res) => {
```

Comment:
These path changes is a very big change. It is although very likely that it is unused. However we should not be modifying paths so quickly and without lots of notice and giving backwards support just by adding the function there too.

Further /apps/appspecifications/:appname/update is not something I would see oftenly. Usually adjustable parameters are at the end so /apps/updatetolatestspces/:appname makes more sense or something like that

Reply:
The update was introduced on latest FluxOs and it's not yet on documentation. Not a issue to put it on a dedicated api call. Regarding the url for it, whatever you guys thinks make sense.